src/SegmentedArc.js

```diff
@@ -167,7 +169,8 @@ export const SegmentedArc = ({
             rangesTextStyle,
             capInnerColor,
             capOuterColor,
-            alignRangesWithSegments
+            alignRangesWithSegments,
+            emptyColor
```

Comment:
This doesn't needed to be added to the provider

Reply:
Removed! Thank you.